src/styles/stopwatch.scss

```diff
@@ -0,0 +1,36 @@
+.clock {
+  width: $clock-size;
+  height: $clock-size;
+
+  @include position-value;
+  position: absolute;
```

Comment:
<img width="928" alt="image" src="https://user-images.githubusercontent.com/102411130/232693957-d15af906-a8b3-4861-97df-c9a0f7f7e001.png">

Reply:
yeah,forgot about that)

Reply:
not fixed https://github.com/mate-academy/layout_stop-watch/pull/2321/files#r1169569301
you should move external styles to bem element. you can use mix(refer to checklist examples)